### PR TITLE
Use isLoading as a separate state property

### PR DESCRIPTION
**Pull request by iloire**

### CODE REVIEW

File: __tests__/index.tsx [hunk 1]
```diff
@@ -76,7 +76,20 @@ describe('VideoRenderer', () => {
   });
 
   describe('state', () => {
```
[the remark this answers is not in the document]
Comment (iloire) [reply __tests__/index.tsx]: I agree, but I didn't find a way to do it easily... maybe you have some additional ideas?

[the remark this answers is not in the document]
Comment (zzarcon) [reply __tests__/index.tsx]: Done it!

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-video-renderer",
-  "version": "1.1.2",
```
Comment (zzarcon) [inline package.json]: We broke the public api with **VideoStatus** type, hence it's major change hehe

Comment (iloire) [reply package.json]: yeap 👍

**Pull request merged**